libbeat/scripts/generate_fields_docs.py

```diff
@@ -90,26 +89,28 @@ def fields_to_asciidoc(input, output, beat):
 
     sections = docs["sections"]
 
+    # Add default section
+    sections["beat"] = "Beat"
+
     # Check if sections is define
     if sections is None:
         print "No sections are defined in fields.yml. fields.asciidoc cannot be generated."
         return
 
-    for doc, _ in sections:
-        output.write("* <<exported-fields-{}>>\n".format(doc))
-    output.write("\n")
 
-    for value in sections:
+    for key in sorted(sections):
+        output.write("* <<exported-fields-{}>>\n".format(key))
+    output.write("\n")
 
-        doc = value[0]
-        name = value[1]
+    for key in sorted(sections):
```

Comment:
This means they will always be documented alphabetically rather than the order we define it, right? I'm not sure if in certain cases we want to control the order in which the sections show up in docs. cc @dedemorton.

Another aprroach would be make the global yaml sections an array, so the order in which they show up in `fields.yml` is the order you get in the docs. And just remove the `sections` part of `fields.yml`.